nifi-ngsi-bundle/nifi-ngsi-processors/src/main/java/org/apache/nifi/processors/ngsi/NGSIToCKAN.java

```diff
@@ -291,7 +280,8 @@ public void aggregate(Entity entity, long creationTime, String dataModel) {
             getLogger().debug("DCAT metadata: {}" , dcatMetadata);
 
             for (Entity entity : entities) {
-                final String pkgName = ckanBackend.buildPkgName(fiwareService,entity,dataModel,enableEncoding,enableLowercase,ngsiVersion,dcatMetadata);
+                final String pkgTitle = ckanBackend.getDataFromRelationshipDetails(entity, "title");
```

Comment:
this is not related to CKAN, the `getDataFromRelationshipDetails` method should go elsewhere (in `NGSIUtils`?).